src/screens/AirdropDashboard/components/WalletBalanceCircle.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
```

Comment:
it should be a global component as it used on the dashboard and the airdrop details screen after the airdrop